v1/api/swagger/paths/allocation@server.yaml

```diff
@@ -2,7 +2,7 @@ get:
   tags:
     - allocation
   summary: Get servers allocation
-  description: Returns an array of allocation information for each server instance type used by each PCE service. If siteID is present, the information returned is specific to that site ID, otherwise the allocation information for all sites is returned.
+  description: Returns an array of allocation information for each server instance type used by each PCE service. If siteID is present, the information returned is specific to that site ID. If the allocation data is needed for a specific project, then project header must be present in the request. In other case, the allocation information for the tenant is returned.
```

Comment:
Use paragraph? That would make it easy to read. Same with volume.

description: | Returns an array of allocation information for each server instance type. Each instance type's allocation information is further grouped by service type.
When using a Metal token, the value in the 'Membership' header determines the scope of the response, i.e., Project or Hoster. However, the default scope is Tenant/Hoster when using the GL IAM token. If the allocation data is needed for a specific project, then the 'Project' header must be present in the request.
If the 'siteID' query parameter is present, the information returned is specific to that site ID.